src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -0,0 +1,33 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { palette, semanticColors } from '../../util/colors';
+import { mergeStyleSets } from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+import { MessageBarType } from './types';
+
+export interface MessageBarStyleProps {
+  type: MessageBarType;
+}
+
+const borderColors = {
+  [MessageBarType.WARNING]: semanticColors.warningBackground,
+  [MessageBarType.ERROR]: semanticColors.errorBackground,
```

Comment:
What do you think of using these semantic names?   I have a tiny hesitation that `background` is misleading, but I think I can live with it.

Reply:
looks good to me

Reply:
we can adjust later if it makes sense